Clarify local test steps in README.md (#20)
os-autoinst/openqa-logwarn: README.md

### Rules for commits

> * Every commit is checked by [Travis CI](https://travis-ci.com/) as soon as

Reviewer: this is outdated in the meantime same as the first line referencing travis CI

Reviewer 2: It is true but also not related to the change suggested here so I would rather change this in a different PR instead of demanding to extend this one :)

Author: I proposed #21 for this

> * Every commit is checked by [Travis CI](https://travis-ci.com/) as soon as
>   you create a pull request.
>   You can locally run `./test_logwarn` or `prove test_logwarn` to run the tests yourself.

Reviewer: please wrap lines consistently at 80 characters.

But also I consider the phrasing not so good. It says "you can run … to run". How about just

```suggestion
 * Every commit is checked by
 [CI](https://github.com/os-autoinst/openqa-logwarn/actions) as soon as you
 create a pull request but you *should* run tests locally.
```

As the usage section already explains how to call tests so please don't repeat the usage in the rules for commit

Reviewer 2: I honestly think the previous phrasing was fine and maybe should just be extended with the additional `prove test_logwarn`

Author: For perspective, I actually read this as a checklist, especially as a new contributor. And my brain parsed "please test manually if you know how". **Usage** is not where I'd look at all.

Reviewer 2: maybe we could link here to the **Usage** section to avoid duplication but also have it included in the "checklist"?

Author: Done as suggested and expanded the existing section a bit

### Rules for commits

* Every commit is checked by [GitHubActions](https://docs.github.com/en/actions)
  as soon as you create a pull request but you *should* test locally, see the

>   [Usage](#usage) section for details.

Reviewer 2: you mean **Testing** section here, right?
